src/Mod/Assembly/App/AssemblyObject.cpp

```diff
@@ -111,6 +111,39 @@ void printPlacement(Base::Placement plc, const char* name)
         angle);
 }
 
+bool isLink(App::DocumentObject* obj)
```

Comment:
When using global functions they should either be made static or put into anonymous namespace as otherwise weird linking problems can occur if there is a name clash.